internal/addrs/remove_endpoint.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package addrs
+
+import (
+	"github.com/hashicorp/hcl/v2"
+	"github.com/opentofu/opentofu/internal/tfdiags"
+)
+
+// RemoveEndpoint is to ConfigMoveable what Target is to Targetable:
+// a wrapping struct that captures the result of decoding an HCL
+// traversal representing a relative path from the current module to
+// a removable object. It is very similar to MoveEndpoint.
+//
+// Its purpose is for the "from" address in a "removed" statement
+// in the configuration.
```

Comment:
this bit of the comment doesn't make sense. Is it copied from the move stuff?

Reply:
No, sorry, it's just poor English. Fixed it.